assets/_scss/core/_variables.scss

```diff
@@ -2,7 +2,7 @@
 $em-base:             10px;
 $base-font-size:      rem(17px);
 $small-font-size:     rem(14px);
-$lead-font-size:      rem(22px);
+$lead-font-size:      rem(20px);
```

Comment:
@maya the visuals of the site? @mollieru and I decided that if we have serif headings we would have serif lead text and that's what we're doing for our default, so I think that 20px should stay. @mollieru thoughts?